share/wake/lib/system/io.wake

```diff
@@ -68,6 +69,7 @@ export def installAs (dest: String) (file: Path): Result Path Error =
         sdest, Nil
     makeExecPlan cmd inputs
     | setPlanEnvironment Nil
+    | setPlanLabel "installAs: {dest}"
```

Comment:
This one might want to record the source file as well.  Also, for `targz` and `claimFileAsPathIn`, you've used slightly more readable "job names", and I'd expect this to be `install-as:` to follow their pattern.